torch/export/dynamic_shapes.py

```diff
@@ -993,7 +1140,7 @@ def root_value():
 
     def update_symbols(path, tensor, shape):
         def _create_static_dim(tensor, i, value):
-            return _StaticDim(str(value), (int,), {"value": value})
+            return _StaticDim(f"{value}_{id(tensor)}_{i}", (int,), {"value": value})
```

Comment:
having the same name for all shapes of a certain value was creating too many unnecessary equality constraints, I'm switching to avoid it

Reply:
Noo, there's some code to clean up these fake dynamic constraints before produce_guards(), it just converts them into static constraints

Reply:
Ok turns out it does, gonna revert this change...